docroot/themes/custom/mass_theme/overrides/js/device.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file
+ * Adds a localStorage item that provides a unique identifier for the user's device while on the site.
+ */
+
+(function ($) {
+  'use strict';
+
+  Drupal.behaviors.massgovDevice = {
+    attach: function (context) {
+      once('massgovDeviceId', 'html', context).forEach(function() {
+        if (window.crypto && localStorage.getItem('massgovDeviceId') === null) {
+          localStorage.setItem('massgovDeviceId', window.crypto.randomUUID());
+        }
+      });
+    }
+  };
+})(jQuery);
```

Comment:
```suggestion
(function (Drupal, once) {
  Drupal.behaviors.massgovDevice = {
    attach: function (context) {
      once('massgovDeviceId', 'html', context).forEach(function() {
        if (window.crypto && localStorage.getItem('massgovDeviceId') === null) {
          localStorage.setItem('massgovDeviceId', window.crypto.randomUUID());
        }
      });
    }
  };
})(Drupal, once);
```

jQuery not needed here :smile_cat:

Reply:
@justafish Ah, thanks! Old habits die hard